docs/addons/aws-for-fluent-bit.md

```diff
@@ -0,0 +1,22 @@
+# AWS for Fluent Bit
+
+Fluent Bit is an open source Log Processor and Forwarder which allows you to collect any data like metrics and logs from different sources, enrich them with filters and send them to multiple destinations.
```

Comment:
Suggest rewording as "Fluent Bit is an open source <strike>L</strike>log processor and <strike>F</strike>forwarder which allows you to collect <strike>any</strike> data  such as metrics and logs from different sources... "